lib/run.rb

```diff
@@ -8,10 +8,17 @@
 require "bot"
 require "configuration/jira"
 
+def get_array_from_env(key)
+  JSON.parse(ENV[key])
+rescue StandardError
```

Comment:
Which error are you expecting here? We should only rescue these errors instead of `StandardError`.

Reply:
To be honest the `StandardError` error type just came from Rubocop's autocorrection of the original line `JSON.parse(ENV[key]) rescue []`
I guess we would expect a `TypeError` if the environment variable for some reason isn't set? As it would then be trying to parse `nil`, and the console gives me
```ruby
[1] pry(main)> JSON.parse nil
TypeError: Nil is not a valid JSON source.
```
for this. Is this reasoning correct? Or could there be other errors?